### DISCUSSION TITLE
option to save time series, not just spectrum?

### DISCUSSION THREAD

**Discussion by jbeale1** [Open-Gamma-Project/Gamma-MCA#217] [Ideas]
Gamma MCA PWA apparently saves the number of received counts per second, in order to display the plot from the "Show Evolution" button. If there is already way to export the count rate vs time as a CSV file, I have not found it. This may not be the usual purpose of a gamma spectrometer, but can be of interest for monitoring variations in background level. Attached plot covers just under 5 days;  variability may be related to precipitation.
![2024-01-19_Bkgnd-NaI-5Day](https://github.com/OpenGammaProject/Gamma-MCA/assets/7452231/5291ee96-ba79-4063-a43e-395e0a4a1bee)

**Comment by NuclearPhoenixx** [comment]
Oh yeah that shouldn't be hard to do at all. Sounds like an interesting use case. I'll have a look at it in the next couple of days. In the mean time you can at least save the plot as an image.

What you can also do is dump all the data from the device into a file via some other serial terminal software (like PuTTY) as it comes in. If you output events, and not the whole spectrum, the number of events per line will then be your cps. So you can load that file into a Python script or something and get a similar time evolution with much better resolution. Just ask ChatGPT if you're unsure how to do that.

**Reply by NuclearPhoenixx** [reply]
Update: It was pretty easy to do indeed. Have fun! https://github.com/OpenGammaProject/Gamma-MCA/pull/218
